temoa/extensions/unit_commitment/core/data_puller.py

```
@@ -0,0 +1,204 @@
+            'online_cap': online,
+            'start_cap': started,
+            'stop_cap': stopped,
+            'units': unit_prop.get_capacity_units(uci.t) if unit_prop else None,
+        }
+        records.append(row)
+    try:
+        writer.connection.execute(
+            'DELETE FROM output_unit_commitment WHERE scenario == ?', (scenario,)
+        )
+    except sqlite3.OperationalError:
+        pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(sqlite3.OperationalError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(sqlite3.OperationalError): ...`

(SIM105)